site/app/views/NavigationView.php

```diff
@@ -139,6 +139,29 @@ public function showGradeables($sections_to_list) {
         <tbody id="{$lower_title}_tbody">
 HTML;
             foreach ($gradeable_list as $gradeable => $g_data) {
+        
+                if ($g_data->getActiveVersion() == -1){
+                    $submission_status = array(
+                        "SUBMITTED" => "<em style='font-size: .8em;'>NOT SUMBITTED</em><br>",
+                        "AUTOGRADE" => ""
+                    );
+                }
+                else{
+                    if ($g_data->getTotalNonExtracreditAutograderPoints() == array()){
+                        $submission_status = array(
+                            "SUBMITTED" => "<em style='font-size: .8em;'>SUMBITTED</em><br>",
+                            "AUTOGRADE" => ""
+                        );  
+                    }
+                    else{
+                        $autograde_points_earned = $g_data->getGradedAutograderPoints(); 
+                        $autograde_points_total = $g_data->getTotalNonExtracreditAutograderPoints();
```

Comment:
The autograde total that would be displayed to students should be the `$g_data->getTotalNonHiddenNonExtraCreditPoints()` (rename that to `getTotalAutograderNonHiddenNonExtraCreditPoints()`) as hidden points are always hidden from the student.